### PR TITLE
Feaure/go coverage/sterling

### PR DESCRIPTION
**Pull request by sdreyer**
Just basic coverage with codecov. Will add formatting and local coverage to separate branch

### CODE REVIEW

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      run:
+        working-directory: ./ff/serving
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
```
Comment (Sami1309) [inline .github/workflows/coverage.yml:17]: some packages use 1.17

File: .github/workflows/coverage.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        run: pip install grpcio-tools
+
+      - name: Install Protobuf
+        run: sudo snap install protobuf --classic
+
+      - name: Setup Proto
+        run: ./gen_grpc.sh
+
+      - name: Build
+        run: go build
```
Comment (Sami1309) [inline .github/workflows/coverage.yml:29]: some builds are failing in the github actions, should it be sure to terminate the whole job if this happens?

**Review by Sami1309 — DISMISSED** [review]
looks good just some questions

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (sdreyer) [reply .github/workflows/coverage.yml:17]: Will update

File: .github/workflows/coverage.yml (hunk 2 above)
Comment (sdreyer) [reply .github/workflows/coverage.yml:29]: Confirmed that any step that fails will cancel entire job. The exception right now is Go tests do to upcoming rework

**Pull request merged**